src/ipahealthcheck/core/core.py

```diff
@@ -276,6 +277,11 @@ def run_healthcheck(self):
                 except Exception as e:
                     logger.error("Unable to initialize %s: %s", name, e)
                     continue
+            if hasattr(registry, 'ca_configured'):
+                ca_configured = registry.ca_configured
+            if 'pkihealthcheck' in name and ca_configured is False:
```

Comment:
I think using `not ca_configured` instead of `ca_configured is False` would be better.

Reply:
The reason for this is that both None and False are True for not ca_configured. I suppose I can default to assuming that ca_configured is True instead of None.

Reply:
No, since this is in core this would likely break pkihealthcheck altogether as it would never be loaded. I think the original code is correct, I'll add some comments to clarify it.

Reply:
OK, understood :)